packages/cli/src/Server.ts

```diff
@@ -559,6 +563,20 @@ class Server extends AbstractServer {
 
 		this.app.use(`/${this.restEndpoint}/variables`, variablesController);
 
+		// ----------------------------------------
+		// Version Control
+		// ----------------------------------------
+
+		// initialize SamlService if it is licensed, even if not enabled, to
+		// set up the initial environment
+		if (isVersionControlLicensed()) {
+			try {
+				await Container.get(VersionControlService).init();
```

Comment:
Correct me if I'm wrong, but you want o have the service being initialised in 2 different situations:

1. When it's already set up, you just load or
2. When setting up the feature (i.e. connecting a repo) then you init the service

I'm asking because conditional init could lead to issues like requiring a user restart after enabling a license for the feature to work

Reply:
That's true. Same for saml, which I will change as well.